index.js

```diff
@@ -17,7 +17,8 @@
 'use strict';
 
 var OpaqueSpan = require('./lib/opaque-span.js');
-var Logger = require('./lib/logger.js');
+var CloudCommon = require('@google/cloud-diagnostics-common');
```

Comment:
Capitalization here incorrectly suggests that this is a constructor / class name. Use `cloudCommon`, or just `common` instead.